Purchases/FoundationExtensions/DateFormatter+Extensions.swift

```diff
@@ -14,32 +14,62 @@
 
 import Foundation
 
-extension DateFormatter {
+/// A type that can convert from and to `Dates`.
+protocol DateFormatterType {
 
-    static let iso8601SecondsDateFormatter: DateFormatter = {
-        let dateFormatter = DateFormatter()
-        dateFormatter.dateFormat = "yyyy-MM-dd'T'HH:mm:ssZ"
-        return dateFormatter
+    func string(from date: Date) -> String
+    func date(from string: String) -> Date?
+
+}
+
+extension DateFormatter: DateFormatterType {}
+extension ISO8601DateFormatter: DateFormatterType {}
+
+internal extension ISO8601DateFormatter {
```

Comment:
internal's the default, right? Is it necessary to add the keyword here?

Reply:
Yeah, I made it explicit to indicate that this is not meant to be `public`, but I can remove it.